pants_plugins/sendwave/pants_docker/test_sources.py

```diff
@@ -16,9 +15,8 @@
 @pytest.fixture
 def rule_runner() -> RuleRunner:
     rule_runner = RuleRunner(
-        target_types=[PythonLibrary, PythonTests, Files, Resources, RelocatedFiles],
+        target_types=[PythonSourceTarget, FileTarget, ResourceTarget],
```

Comment:
I cleaned up the target types here since it seemed on closer inspection that we weren't actually using `PythonLibrary` or `PythonSourceTarget` in the tests.

Reply:
I added those back I think because we needed the rules that they setup to the rule_runner - even if we weren't using the targets directly we need those rules since they are included transiently by the stuff we are doing